change/@ni-nimble-components-939685fd-01fb-4080-8431-b6aab0c7b29f.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Localize number-field decimal separator. Breaking change: users in locales with a comma as the decimal separator will need to input a comma rather than a period in the nimble-number-field component.",
```

Comment:
I think we should hold off merging this until after the release branch for 2025-07 is created